src/officehours_ui/static/css/main.css

```diff
@@ -174,11 +174,11 @@ h3, .h3 {
 }
 
 a:not(.btn), a:link:not(.btn), a:visited:not(.btn) {
-	color: #0075bc;
+	color: #004a8b;
 }
 
 a:hover {
-	color: #0065a3;
+	color: #004a8b;
 }
```

Comment:
@teshsopher, I modified the existing link color for non-button links to be a darker shade of blue, resulting in a 8.92 contrast ratio with white and a 7.21 contrast ratio with the light blue Bootstrap color seen in the background for `info` alerts (e.g. Message of the Day, "This is your queue, you can manage it"). Does this resolve those issues you raised? Do you want to tweak the hex value at all? I'm a little wary of making this so dark globally, since it may be getting too close to black and may make links not easily distinguishable (without the user hovering). It would be nice to keep the link color consistent across the board. Would AA level be enough in this situation?

Reply:
Here's how it looks: 
![image](https://user-images.githubusercontent.com/11338926/97454567-4b170080-190d-11eb-816f-2f0d40be599d.png)

A different approach would be to lighten the shade of blue in `info` alerts.

Reply:
Meeting AAA requirements is just an ideal! In this case, AA is good enough, and having the link text be too close to black can definitely be a problem, like you mentioned. Unfortunately, it looks like, for the blue background, the darker color works best, or we would have to change the blue background to something lighter like F0F9F9, but then we risk it not being visible against the white. 

In the case of the darker text, it may be helpful to underline links that are in the alert/message boxes so that they stand out more and look more like clickable links.

Reply:
K, let me see what I can do. I'm gonna bump the link color down a bit so it's AA but still looks link-y. I'll also look into the underlining on alerts, though in an ideal world I'd prefer if links looked/behaved the same everywhere in the application.

Reply:
Okay, I switched to `#0060ba` for links, which puts us at 6.2 contrast with white, and 5.02 with the `info` background color.

Reply:
@teshsopher, do we think the color change will be sufficient? We can certainly add the underline with alerts using CSS, but I do think there is some benefit to link behavior being consistent throughout. I'm looking at how it would be if the  underline was everywhere, but that seems a little distracting in places like the breadcrumbs and queue list table.

Reply:
The link color is very similar to the regular text, so it may be confusing with just the color change. For the sake of clarity, adding some other indicator would be good in this case. However, I do understand your concern with having the links be consistent throughout, and it may be worth underlining text when it's in a body of text, such as the Message of the Day or in the "Joining the meeting" box, and not underlining the breadcrumbs. For example, [the BlueJeans page](https://its.umich.edu/communication/videoconferencing/blue-jeans) on the ITS site uses a similar pattern to help with this type of issue. I agree that it can be a little overwhelming visually if we have too much text grouped together or several links in a small space, so it's definitely something that can use some refining.

Reply:
Okay, let me give this a shot.

Reply:
Okay, @teshsopher, I think I did what you requested here: https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/191/commits/fccc764128fd864c2bb4bb2dd71c7443dbca61d9

I think this is a good solution for now. I'm going to merge this so you can start taking a look in QA.